### ISSUE TITLE
Could not find boost_python

### ISSUE THREAD

**Issue by potuz** [stefantalpalaru/gentoo-overlay#24]
I fail to build with 

```
...
-- Found PythonLibs: /usr/lib/libpython2.7.so (found suitable version "2.7.15", minimum required is "2.6") 
Using libclang to provide semantic completion for C/C++/ObjC
CMake Error at /usr/share/cmake/Modules/FindBoost.cmake:2044 (message):
  Unable to find the requested Boost libraries.

  Boost version: 1.65.0

  Boost include path: /usr/include

  Could not find the following Boost libraries:

          boost_python

  Some (but not all) of the required Boost libraries were found.  You may
  need to install these additional Boost libraries.  Alternatively, set
  BOOST_LIBRARYDIR to the directory containing Boost libraries or BOOST_ROOT
  to the location of Boost.
...
```

and I have these
```
$ locate boost_python
/usr/lib64/libboost_python-2.7-mt.so
/usr/lib64/libboost_python-2.7.so
/usr/lib64/libboost_python-2.7.so.1.65.0
/usr/lib64/libboost_python-3.5-mt.so
/usr/lib64/libboost_python-3.5.so
/usr/lib64/libboost_python-3.5.so.1.65.0
/usr/lib64/libboost_python-3.6-mt.so
/usr/lib64/libboost_python-3.6.so
/usr/lib64/libboost_python-3.6.so.1.65.0
```

**Comment by stefantalpalaru** [comment]
What package are you trying to build?

**Comment by potuz** [comment]
Oops! I apologize: it's

youcompleteme-20180121-r1.ebuild

**Comment by stefantalpalaru** [comment]
Fixed in youcompleteme-20180820 and youcompleteme-99999999-r4.
